backend/capellacollab/tools/models.py

```diff
@@ -240,6 +240,70 @@ class Resources(core_pydantic.BaseModelStrict):
         default=MemoryResources(),
         description="Configuration about the amount of memory that sessions can use.",
     )
+    usernames: list[str] | None = pydantic.Field(
+        default=None,
+        description="List of usernames, which are allowed to use this resource profile.",
+    )
+
+
+class Resources(core_pydantic.BaseModelStrict):
+    profiles: dict[str, ResourceProfile] = pydantic.Field(
+        default={"default": ResourceProfile()},
```

Comment:
```suggestion
    profiles: dict[str, ResourceProfile] = pydantic.Field(
        default={"default": ResourceProfile()},
```

```suggestion
    profiles: dict[str, DefaultResourceProfile | AdditionalResourceProfile] = pydantic.Field(
        default={"default": DefaultResourceProfile()},
```

The goal is to remove the `usernames` attribute from the default profile. It doesn't make munch sense and might lead to confusion. We just need to make sure that DefaultResourceProfile is not used as type for additional profile via validation. 

Alternatively, I'd propose to have the default_profile as separate key and add additional_profiles as list. This would have a better and clear type separation.